### PR TITLE
Improved the design of the Recipe Form

### PR DESCRIPTION
**Pull request by Janicke01**

### CODE REVIEW

**Review by u22633601 — APPROVED** [review]

**Review by u22633601 — CHANGES_REQUESTED** [review]
Just fix the merge conflicts with main

File: lib/screens/home_screen.dart
Comment (Janicke01) [inline lib/screens/home_screen.dart:1]: This was just changes to the background and text color.

File: lib/screens/add_recipe_screen.dart
Comment (Janicke01) [inline lib/screens/add_recipe_screen.dart:1]: Everything looks in order, it doesn't look if there are any merge conflicts on my side that could cause issues on the backend. Just have a look @u22633601

**Review by Janicke01 — COMMENTED** [review]
I had a look through everything and everything seems to be in order, does not like anything could cause issues on the back end side of things.

**Review by u22633601 — APPROVED** [review]

**Pull request merged**